jicofo-common/src/main/kotlin/org/jitsi/jicofo/xmpp/jingle/JingleSession.kt

```diff
@@ -88,11 +105,14 @@ class JingleSession(
     fun terminate(
         reason: Reason,
         message: String?,
-        sendTerminate: Boolean
+        /** Whether to send a session-terminate IQ, or only terminate the session locally. */
+        sendIq: Boolean
     ) {
-        logger.info("Terminating session with $remoteJid, reason=$reason, sendTerminate=$sendTerminate")
+        logger.info("Terminating session with $remoteJid, reason=$reason, sendIq=$sendIq")
+        if (state != State.ACTIVE) logger.error("Sending session-terminate for session in state $state")
```

Comment:
Can this happen?  Do you definitely want to send session-terminate if you hit this, rather than bailing out?

Reply:
I think it can happen with a session in PENDING (i.e. session-initiate sent, session-accept not received (yet)) and I think in this case it makes sense to send it. I'm not sure what the previous behavior was, it would probably not send anything as it didn't have a reference to the session. 

I cleaned up the log messages and suppressed sending the IQ if the session is already ENDED. WDYT?